docs/fundamentals/code-analysis/configuration-options.md

```diff
@@ -94,7 +94,7 @@ The following table shows the different rule severities that you can configure f
 
 - **Category of rules**
 
-  To set the default rule severity for a category of rules, use the following syntax.
+  To set the default rule severity for a category of rules, use the following syntax. (Notice that the prefix is slightly different when disabling via category. It's `dotnet_analyzer_diagnostic` instead of `dotnet_diagnostic`.)
```

Comment:
If I recall correctly, setting a severity for a whole category doesn't really affect every rule in the category. It used to affect only enabled-by-default rules.

It would be worth documenting if that still holds. @mavasani Are you able to confirm?

Reply:
Yes that is still correct. We do have a note about this in category based severity configuration.